CMake/maptk-external-kwiver.cmake

```diff
@@ -0,0 +1,66 @@
+# kwiver External Project
+
+message(STATUS "Configuring external KWIVER")
+
+list(APPEND MAPTK_DEPENDENCIES kwiver)
+
+ExternalProject_Add(kwiver
+  DEPENDS ${KWIVER_DEPENDENCIES}
+  PREFIX ${MAPTK_BINARY_DIR}
+  SOURCE_DIR ${MAPTK_EXTERNAL_DIR}/kwiver
+  BINARY_DIR ${MAPTK_EXTERNAL_DIR}/kwiver-build
+  STAMP_DIR ${MAPTK_STAMP_DIR}
+  GIT_REPOSITORY "https://kwgitlab.kitware.com/computer-vision/KWIVER.git"
```

Comment:
This needs to be updated to point to Github once the KWIVER branch lands in master